### DISCUSSION TITLE
Android Studio

### DISCUSSION THREAD

**Discussion by DevVigilante** [Sketchware-Pro/Sketchware-Pro#784] [Q&A] [answered]
Today I exported a new Sketchware proj. to Android Studio everything works perfectly but local lib is not added.
I don't have much knowledge about gradle and Android Studio please can you guide me how to add local lib which I used I Sketchware??

**Comment by DevVigilante** [comment]
Some local lib I added is not available on Internet just created my self and some of theme I downloaded from Internet manually and they are not getting added.

**Reply by PranavPurwar** [reply]
move them in app/libs folder of project

**Reply by DevVigilante** [reply]
> move them in app/libs folder of project

Tried but still gives error.

**Comment by PranavPurwar** [comment, marked as answer]
add this in build.gradle
```
implementation fileTree(dir: "libs", include: ["*.jar"])
```

**Reply by DevVigilante** [reply]
app
 | - - src
 | - - libs/LibName/Lib files(Jar,Aar,with
 | - - src
 | - - build.gradle

**Reply by DevVigilante** [reply]
> add this in build.gradle
> 
> ```
> implementation fileTree(dir: "libs", include: ["*.jar"])
> ```

Already included this line but I think the path is wrong....

**Reply by PranavPurwar** [reply]
you need to add jar in libs folder not rhe lib folder bruh

**Reply by PranavPurwar** [reply]
now close this discussion

**Reply by DevVigilante** [reply]
> now close this discussion

I thought there is no option for that.

**Comment by DevVigilante** [comment]
I fixed it.

**Reply by j-px** [reply]
This isn't an answer as it doesn't tell how you fixed the issue.

**Reply by DevVigilante** [reply]
I earlier don't found any solution and I implemented using `implementation 'lib:module:version'` but as jbk you gave solution on discord to zip all files in `.sketchware/local_lib/{libname}` in aar format(excluding dex) and this also fixed my issue.
